### DISCUSSION TITLE
Having an issue with Portable-Ruby

### DISCUSSION THREAD

**Discussion by RandallWyatt** [Homebrew/discussions#292] [Everyday usage]
Hello All, 

I am having an issue with portable ruby. When I try to `brew install` anything, I get a message that says this:

`==> Downloading https://homebrew.bintray.com/bottles-portable-ruby/portable-ruby-2.6.3_2.yosemite.bottle.tar.gz
Already downloaded: Users/{my user}/Caches/Homebrew/portable-ruby-2.6.3_2.yosemite.bottle.tar.gz`
`Error: Checksum mismatch.`
`Expected: {the checksum expected}`
`Actual: {the checksum found}`
`Archive: /Users/{my user}/Library/Caches/Homebrew/portable-ruby-2.6.3_2.yosemite.bottle.tar.gz`
`To retry an incomplete download, remove the file above.`
`Error: Failed to install Homebrew Portable Ruby (and your system version is too old)!`

I have done ```rm -rf /Users/{my user}/Library/Caches/Homebrew/portable-ruby-2.6.3_2.yosemite.bottle.tar.gz``` several times and I still get that error. Suggestions?

**Comment by jvitoramorim** [comment]
Hey man, 

I was facing the same issue. Turned out to be my company proxy. hope this helps.
